website/source/docs/guides/acl-migrate-tokens.html.md

```diff
@@ -0,0 +1,415 @@
+---
+layout: "docs"
+page_title: "ACL Token Migration"
+sidebar_current: "docs-guides-acl-migrate-tokens"
+description: |-
+  Consul 1.4.0 introduces a new ACL system with improvements for the security and
+  management of ACL tokens and policies. This guide documents how to upgrade
+  existing (now called "legacy") tokens after upgrading to 1.4.0.
+---
+
+# ACL Token Migration
+
+Consul 1.4.0 introduces a new ACL system with improvements for the security and
+management of ACL tokens and policies. This guide documents how to upgrade
+existing (now called "legacy") tokens after upgrading to 1.4.0.
+
+Since the policy syntax changed to be more precise and flexible to manage, it's
+necessary to manually translate old tokens into new ones to take advantage of
+the new ACL system features. Tooling is provided to help automate this and this
+guide describes the overall process.
+
+~> **Note:** **1.4.0 retains full support for "legacy" ACL tokens**
+so in-place upgrade is safe and existing tokens will continue to work in the
```

Comment:
Maybe remove the "so" and make "upgrade" plural since we are saying "tokens"  = "...tokens**, in-place upgrades are safe...."

Reply:
Hmm maybe highlights an ambiguity here but I was referring to the whole cluster upgrade from 1.3.x -> 1.4.0 being safe.

Perhaps:

> 1.4.0 retains full support for "legacy" ACL tokens so upgrades from Consul 1.3.0 are safe. Existing tokens will ...